### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 16

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#342] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152325258-1b5cb678-1102-4861-909a-13a051cf6db0.png)

**Comment by i0nut02** [comment]
<img width="871" alt="Schermata 2022-02-05 alle 11 28 39" src="https://user-images.githubusercontent.com/99051485/152638387-9ffae02b-4b8f-4f6d-8987-27960860f90f.png">

**Comment by CiottoloMaggico** [comment]
![immagine](https://user-images.githubusercontent.com/79638284/217812084-c0cac4e9-28a8-4d6c-b319-ad7fef014219.png)

**Comment by francesco-alberghina** [comment]
Un esempio possibile di risoluzione:
![immagine](https://github.com/sapienzastudentsnetwork/mmi2122/assets/68157574/252ee5ad-f5cf-4a15-bfde-8d3046905b5e)
